classes/codebench.bbclass

```diff
@@ -41,6 +53,20 @@ python codebench_check () {
     else:
         d.setVar('EXTERNAL_TARGET_SYS', triplets[0])
         d.setVar('EXTERNAL_TOOLCHAIN', os.path.join(toolchains_path, toolchain_subdir))
+
+python codebench_check () {
+    codebench_path = d.getVar('CODEBENCH_PATH', True)
+    exttc = d.getVar('EXTERNAL_TOOLCHAIN', True)
+    if exttc:
+        if codebench_path:
+            # Explicitly set EXTERNAL_TOOLCHAIN is preferred
```

Comment:
If the EXTERNAL_TOOLCHAIN is invalid, it'll error out, but I can see a warning being useful for the case where both are set and both are valid, so the user is clear on which is being used. Will do.